chapter_appendix-mathematics-for-deep-learning/information-theory.md

```diff
@@ -162,7 +162,7 @@ $$H(S) = \sum_i {p_i \cdot I(s_i)} = - \sum_i {p_i \cdot \log p_i}.$$
 
 By the above examples and interpretations, we can derive the following properties of entropy :eqref:`eq_ent_def`. Here, we refer to $X$ as an event and $P$ as the probability distribution of $X$.
 
-* Entropy is non-negative, i.e., $H(X) \geq 0, \forall X$.
+* If $X$ is discrete, then entropy is non-negative, i.e., $H(X) \geq 0, \forall X$ discrete. (However, if $X$ is continuous, then entropy can be negative.)
```

Comment:
$H(X) \geq 0, \forall X$ discrete. (However, if $X$ is continuous, then entropy can be negative.)
->
$H(X) \geq 0$ for all discrete $X$ (entropy can be negative for continuous $X$).